client/driver/rkt.go

```diff
@@ -764,12 +1030,22 @@ func (h *rktHandle) Exec(ctx context.Context, cmd string, args []string) ([]byte
 	if h.uuid == "" {
 		return nil, 0, fmt.Errorf("unable to find rkt pod UUID")
 	}
+
 	// enter + UUID + cmd + args...
-	enterArgs := make([]string, 3+len(args))
+	// If a pod manifest with multiple apps is used, use the first app for the command:
+	// enter + --<app=appname> + UUID + cmd + args...
```

Comment:
@indradhanush IMO, Having a `TO DO` explanation here like I did in my branch, of some better way to fix this hack will be very helpful. We know this is a hack, and the ideal thing is to let the required app's name be passed in, rather than just using the first app. Skipping that explanation will not help us or them remember that the source of that code needs to be adjusted for it to be flexible when a multiple apps PodManifest is passed in.

Reply:
@kosyfrances Yes this sounds good to me. I'm not very sure why I had removed it back while rebasing. Sorry about that.

Reply:
Please do this

Reply:
Done, but doesn't reflect as outdated here because the lines after 1036 were changed.